crates/rolldown/src/utils/renamer.rs

```diff
@@ -35,12 +34,15 @@ pub struct Renamer<'name> {
 }
 
 impl<'name> Renamer<'name> {
-  pub fn new(symbols: &'name Symbols, _modules_len: usize) -> Self {
+  pub fn new(symbols: &'name Symbols, _modules_len: usize, format: &OutputFormat) -> Self {
+    let manual_reserved =
+      if matches!(format, OutputFormat::Esm) { vec![] } else { vec!["exports"] };
```

Comment:
Here should be port from https://github.com/rollup/rollup/blob/master/src/Chunk.ts#L1377-L1394.

Reply:
I have modified it.